<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="3,0">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="frame_ok_cancel">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin" stdset="0">
      <number>0</number>
     </property>
     <item>
      <spacer name="spacer_ok_chanel">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Отменить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="frame_settings">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="11" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBox_reinstall_modpack">
        <property name="text">
         <string>Переустоновить сборку</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="SomSlider" name="horizontalSlider_memory">
        <property name="maximum">
         <number>1024</number>
        </property>
        <property name="singleStep">
         <number>1024</number>
        </property>
        <property name="pageStep">
         <number>1024</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>1024</number>
        </property>
       </widget>
      </item>
      <item row="14" column="0" colspan="4">
       <widget class="QLabel" name="label_account">
        <property name="text">
         <string>Аккаунт</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="4">
       <widget class="QLabel" name="label_memory">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Память</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="4">
       <widget class="Line" name="line_first">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="16" column="0" colspan="4">
       <widget class="QPushButton" name="pushButton_exit_account">
        <property name="text">
         <string>Выйти из аккаунта</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="4">
       <widget class="QLabel" name="label_modpak">
        <property name="text">
         <string>Сборка</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QToolButton" name="toolButton_java_path">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QLabel" name="label_max_memory">
        <property name="text">
         <string>999999999</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QLineEdit" name="lineEdit_java_path"/>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QLabel" name="label_game_path">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Путь игры</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLineEdit" name="lineEdit_game_path"/>
      </item>
      <item row="1" column="3">
       <widget class="QToolButton" name="toolButton_game_path">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="4">
       <widget class="Line" name="line_second">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="13" column="0" colspan="4">
       <widget class="Line" name="line_third">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_min_memory">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>9999</string>
        </property>
       </widget>
      </item>
      <item row="15" column="0" colspan="2">
       <widget class="QLabel" name="label_account_name">
        <property name="text">
         <string>account_name</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="4">
       <widget class="QLabel" name="label_memory_count">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>memory_count</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="QLabel" name="label_java_path">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Путь к java</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_change_mods">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Изменить моды</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SomSlider</class>
   <extends>QSlider</extends>
   <header>../../../QObjects/SomSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>released()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
